dali/test/python/test_pipeline.py

```diff
@@ -660,7 +660,7 @@ def iter_setup(self):
     for i in range(batch_size):
         NonFused_pipe_out_cpu = nonfused_pipe_out[0].as_cpu()
         fused_pipe_out_cpu = fused_pipe_out[0].as_cpu()
-        assert(np.sum(np.abs(NonFused_pipe_out_cpu.at(i)-fused_pipe_out_cpu.at(i)))==0)
+        assert(np.sum(np.abs(nonFused_pipe_out_cpu.at(i)-fused_pipe_out_cpu.at(i)))==0)
```

Comment:
now the variable name does not match the actual name in line 661. Also, lets name it `non_fused_pipe_out_cpu`